scss/atoms/notifications/_mixins.scss

```diff
@@ -0,0 +1,18 @@
+/*
+* ==========================================================================
+* File: _mixins.scss
+* For : Notifications
+* ==========================================================================
+*/
+
+@mixin notification($type) {
+	border-radius: map-get($notifications-element, border-radius);
+	box-shadow: map-get($notifications-shadows, box-shadow-base);
+	margin: map-get($notifications-element, margin);
+	letter-spacing: map-get($notifications-element, letter-spacing);
+	font-weight: map-get($notifications-element, font-weight);
+	padding: map-get($notifications-element, padding);
+	font-size: map-get($notifications-element, font-size);
+	background: bg($type);
```

Comment:
See: 
https://github.com/jquery/css-chassis/blob/master/scss/atoms/buttons/_buttons.scss#L45
and https://github.com/jquery/css-chassis/blob/master/scss/atoms/buttons/_mixins.scss#L12